files-community/Files
# Code Quality: Refactor preview/info pane phase 1

**Resolved / Related Issues**
   Closes #12964...

**Validation**
How did you test these changes?
   1. Open app and enable info pane
   2. Confirm that it works to switch between details & preview via the tabs as well as the keyboard shortcuts
   3. Confirm that the selected tab is restored even if the info pane is closed and reopened

**Screenshots (optional)**
Add screenshots here.
